code/game/sound.dm

```diff
@@ -5,25 +5,20 @@
 
 	var/turf/turf_source = get_turf(source)
 
-	//allocate a channel if necessary now so its the same for everyone
-	channel = channel || open_sound_channel()
-
- 	// Looping through the player list has the added bonus of working for mobs inside containers
-	var/sound/S = sound(get_sfx(soundin))
-	var/maxdistance = (world.view + extrarange)
-	var/list/listeners = GLOB.player_list
-	if(!ignore_walls) //these sounds don't carry through walls
-		listeners = listeners & hearers(maxdistance,turf_source)
-	for(var/P in listeners)
-		var/mob/M = P
-		if(!M || !M.client)
-			continue
-		var/distance = get_dist(M, turf_source)
-
-		if(distance <= maxdistance)
-			var/turf/T = get_turf(M)
-
-			if(T && T.z == turf_source.z)
+	if (turf_source)
+		//allocate a channel if necessary now so its the same for everyone
+		channel = channel || open_sound_channel()
```

Comment:
shouldn't this be channel |= open_sound_channel() ?

Reply:
No, `|` and `||` are different. It could be `||=` if we had that but we don't.